examples/libf3d/cpp/multi-geom/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ endif()
 # Simple testing
 if(BUILD_TESTING)
   enable_testing()
-  add_test(NAME test_multi-geom COMMAND "$<TARGET_FILE:multi-geom>" "${CMAKE_CURRENT_SOURCE_DIR}/data/")
+  add_test(NAME test_multi-geom COMMAND "$<TARGET_FILE:multi-geom>" "${CMAKE_CURRENT_SOURCE_DIR}/../../data/datasets")
```

Comment:
why changing the location ?

Reply:
To avoid data duplication and use it into python examples tests. But If you prefer I can move python porting to related folders

Reply:
examples should be considered independant from the repo if needed, that is why it has its own data directory.

Reply:
Ok I make the move ! Do you prefer create external python folder for python examples or add each python scripts in the dedicated example folder ?

Reply:
imo `examples/libf3d/cpp` and `examples/libf3d/python` makes sense.